### PR TITLE
Add separator between personal navigation items

### PR DESCRIPTION
**Pull request by dmarcoux**
I had a flash after the stand-up when we talked about this issue.

I don't consider this done as this needs to be fixed:
- [x] Separators are in the default text color (so they aren't like the links they separate)
- [ ] The padding between separators and links has to be consistent

I open this PR to get your suggestions on how to fix these remaining issues

### CODE REVIEW

File: src/api/app/assets/stylesheets/webui2/main.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#personal-navigation ul.nav >li.nav-item > a {
+  padding-left: 0;
+  padding-right: 0;
+}
+
+#personal-navigation ul.nav > li.nav-item:not(:last-child) > a::after {
```
Comment (hennevogel) [inline src/api/app/assets/stylesheets/webui2/main.scss]: and :before? :-)

File: src/api/app/assets/stylesheets/webui2/main.scss [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#personal-navigation ul.nav >li.nav-item > a {
+  padding-left: 0;
+  padding-right: 0;
```
Comment (hennevogel) [inline src/api/app/assets/stylesheets/webui2/main.scss]: There are [bootstrap classes](https://getbootstrap.com/docs/4.1/utilities/spacing/) for this.

**Comment by dmarcoux** [comment]
I just talked with @mdeniz and he will submit a PR including my changes. He's working on the structure on the personal navigation.

File: src/api/app/assets/stylesheets/webui2/main.scss (hunk 1 above)
Comment (dmarcoux) [reply src/api/app/assets/stylesheets/webui2/main.scss]: With `::after`, it reproduces what we have in the old interface.

**Pull request closed without merging**